### DISCUSSION TITLE
Bookmark Navigation

### DISCUSSION THREAD

**Discussion by Zodler** [SubtitleEdit/subtitleedit#8252] [Ideas]
I was going to post that we desperately need a bookmark solution and saw that it is already implemented!

However we need a mechanism to navigate them faster and easier. Like a previous and next button that would jump only between bookmarks.

I see that we can define shortcuts for that. Maybe also 2 buttons that we could use with mouse would be perfect.

**Comment by niksedk** [comment]
The shortcuts will have to do :)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/3af26092-d118-4af2-822e-75d824712007)
